Sources/NIOPosix/System.swift

```diff
@@ -534,14 +567,15 @@ internal enum Posix {
     }
 
     @inline(never)
-    internal static func bind(descriptor: CInt, ptr: UnsafePointer<sockaddr>, bytes: Int) throws {
+    public static func bind(descriptor: CInt, ptr: UnsafePointer<sockaddr>, bytes: Int) throws {
```

Comment:
Can we drop the changes to `public` here? They're very misleading.

Reply:
@Lukasa there were other, newer (for NIOFileSystem etc) functions that used `public`, so I thought I'd make it consistent with the other new stuff